### ISSUE TITLE
Examples do not unwrap TermLogger

### ISSUE THREAD

**Issue by enderger** [Drakulix/simplelog.rs#61]
In the current version of simplelog, it seems that you need to unwrap the option containing a TermLogger when initializing the CombinedLogger

**Comment by John-Nagle** [comment]
Yes. Compiling the example from the web site gives:

```
error[E0599]: no method named `unwrap` found for struct `std::boxed::Box<simplelog::TermLogger>` in the current scope
  --> src/main.rs:11:88
   |
11 |             TermLogger::new(LevelFilter::Warn, Config::default(), TerminalMode::Mixed).unwrap(),
   |                                                                                        ^^^^^^ method not found in `std::boxed::Box<simplelog::TermLogger>`

error: aborting due to previous error
```

**Comment by Drakulix** [comment]
Fixed by #60
